cloc

```diff
@@ -10633,6 +10639,7 @@ sub set_constants {                          # {{{1
     'synon/2e'                     =>   4.21,
     'systemw'                      =>   2.22,
     'tandem access language'       =>   0.88,
+    'TableGen'                     =>   2.00,
```

Comment:
I'm not sure about how to calculate this value, I think `TableGen` is used to generate `C++` code, 2.0 might be reasonable.